suite-native/module-passphrase/src/screens/PassphraseEmptyWalletScreen.tsx

```diff
@@ -0,0 +1,98 @@
+import { DimensionValue } from 'react-native';
+import { useState } from 'react';
+
+import { Screen } from '@suite-native/navigation';
+import { VStack, Card, Text, Image, Button, Box } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+import { PassphraseScreenHeader } from '../components/PassphraseScreenHeader';
+import { EmptyWalletInfoSheet } from '../components/EmptyWalletInfoSheet';
+
+const imageStyle = prepareNativeStyle(() => ({
+    width: 124.45,
+    height: 96,
+    alignSelf: 'center',
+}));
+
+const cardStyle = prepareNativeStyle(utils => ({
+    borderColor: utils.colors.borderElevation0,
+    borderWidth: utils.borders.widths.small,
+}));
+
+const cardContentStyle = prepareNativeStyle(() => ({
+    padding: 54,
+}));
+
+const buttonWrapperStyle = prepareNativeStyle(() => ({
+    width: '100%',
```

Comment:
the width should be the same as the confirm button above acording to design file. Now it is as wide as the whole card.

Reply:
As mentioned in the comment in JSX, this will be done in more detail in a separate issue.